android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/screen/common/ImageBackground.kt

```diff
@@ -0,0 +1,61 @@
+package co.nimblehq.kaylabruce.kmmic.android.presentation.screen.common
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.blur
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+private const val GRADIENT_VAL = 0.2f
+@Composable
```

Comment:
```suggestion
private const val GRADIENT_VAL = 0.2f

@Composable
```
I think we should have an empty line here to separate constant value and the composable function